### DISCUSSION TITLE
VMX2 Case Record Type

### DISCUSSION THREAD

**Discussion by SyndrixLane** [amazon-connect/amazon-connect-salesforce-scv#124] [Q&A] [answered]
Hello,

I am trying to get the VMX2 code to create a case of a specific record type. We use the case object for multiple things. 

I believe this is is the line of code that I need to update in the packager lambda, sub_salesforce_case.py
do_create = sf.create(sobject='Case', data=data)

I also need the code to include RecordTypeId='0123C000000cm9uQAA' for the case object. Can you show me how to properly add that into this line of code?

Thank you

**Comment by dougjaso** [comment, marked as answer]
In sub_salesforce_case.py, after line 74:
`                'OwnerId': sf_agent_id,`
add a new line and paste:
`                'RecordTypeId': '0123C000000cm9uQAA',`

then, after line 86 (will be line 87 after the above edit):
`                'Priority': writer_payload['json_attributes']['vm_priority'],`
add a new line and paste:
`                'RecordTypeId': '0123C000000cm9uQAA',`

Save and deploy. That should get you there.

**Reply by SyndrixLane** [reply]
Thank you, this worked.
